portfolio/src/main/webapp/index.html

```diff
@@ -1,83 +1,78 @@
 <!DOCTYPE html>
-<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
-<div id="niceback">
 <html>
+<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
   <head>
     <meta charset="UTF-8">
     <title>Michael's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-<div id="content">
-      <h1><p3 style="color:white">Michael Peters's Portfolio </p3></h1>
-      <center><img src="/images/IMG_3549.jpg" width="120" height="190"></center>
-      
-      
-      
-
-<div id="greeting-container"></div>
+    <div id="gradientBackground">
+    <div id="content">
+        <h1><p3>Michael Peters' Portfolio </p3></h1>
```

Comment:
I'm not sure if these are working, but only `<p>` tags and `<h1>` to `<h6>` tags exist. The tags you're using here, `<p3>` and `<p1>`, shouldn't be doing anything. Also specifically in this case, it's weird to do `<h1><p>text</p></h1>` because you're telling the html that you want it to be styled as an h1 heading AND as a paragraph, so you should pick one or the other.